cmd/commands/commands.go

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2023 Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	readfile "github.com/open-cmsis-pack/generator-bridge/internal/readFile"
+	stm32cubemx "github.com/open-cmsis-pack/generator-bridge/internal/stm32CubeMX"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// AllCommands contains all available commands for generator-bridge
+var AllCommands = []*cobra.Command{}
+
+var rootCommand *cobra.Command = nil
+
+func GetConfig() *pflag.FlagSet {
+	return rootCommand.PersistentFlags()
+}
+
+// configureInstaller configures generator-bridge installer for adding or removing pack/pdsc
+func configureGlobalCmd(cmd *cobra.Command, args []string) error {
+	verbosiness, _ := GetConfig().GetBool("verbose")
+	quiet, _ := GetConfig().GetBool("quiet")
+	if quiet && verbosiness {
+		return errors.New("both \"-q\" and \"-v\" were specified, please pick only one verboseness option")
+	}
+
+	log.SetLevel(log.InfoLevel)
+	log.SetOutput(cmd.OutOrStdout())
+
+	if quiet {
+		log.SetLevel(log.ErrorLevel)
+	}
+
+	if verbosiness {
+		log.SetLevel(log.DebugLevel)
+	}
+
+	return nil
+}
+
+var flags struct {
+	version bool
+	help    bool
+	inFile  string
+	inFile2 string
+	outPath string
+}
+
+var Version string
+var CopyRight string
+
+func printVersionAndLicense(file io.Writer) {
+	fmt.Fprintf(file, "generator-bridge version %v %s\n", strings.ReplaceAll(Version, "v", ""), CopyRight)
```

Comment:
Copyright is a term/noun and not constructed from copy and right. Therefore camelcasing does not seem right.